### PR TITLE
because i had to investigate shit about random murder with your goddamn gun and it didn't tell me who did what!

### PR DESCRIPTION
**Pull request by silicons**
# 150 HOURS OF TESTING

### CODE REVIEW

**Comment by MrStonedOne** [comment]
\>no `[s]`

**Comment by kevinz000** [comment]
:^)

File: code/game/machinery/porta_turret/portable_turret.dm [hunk 1]
```diff
@@ -1052,15 +1052,15 @@
 	else
 		cooldown = world.time + cooldown_duration
 		warned = FALSE
-		INVOKE_ASYNC(src, /obj/machinery/manned_turret/.proc/volley)
+		INVOKE_ASYNC(src, /obj/machinery/manned_turret/.proc/volley, user)
```
Comment (Cyberboss) [inline code/game/machinery/porta_turret/portable_turret.dm]: WHY IS THIS INVOKE_ASYNC!!

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/machinery/porta_turret/portable_turret.dm (hunk 1 above)
Comment (kevinz000) [inline code/game/machinery/porta_turret/portable_turret.dm]: @Robustin 
@Robustin 
@Robustin 
@Robustin

Comment (Cyberboss) [inline code/game/machinery/porta_turret/portable_turret.dm]: Make it a normal call please

**Comment by kevinz000** [comment]
#27584

**Comment by Robustin** [comment]
Exactly why is this a 150 hours issue and why are you sperging out even harder than usual?

**Pull request closed without merging**